django/forms/models.py

```diff
@@ -1597,6 +1611,7 @@ def to_python(self, value):
 
     def clean(self, value):
         value = self.prepare_value(value)
+        self.validate_non_null_characters(value)
```

Comment:
I would then move this inside `_check_values` when it is looping through the pks
```python
        for pk in value:
            self.validate_non_null_characters(value)
            try:
                self.queryset.filter(**{key: pk})
            except (ValueError, TypeError):
```